viewing_party/party.py

```diff
@@ -1,23 +1,291 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    """ creating movie dictionary and return movies of dictionary """
+
+    # ditionary to store movies
+    movies = {}
+
+    # if any one of paramenters is None, function return None
+    if title == None or genre == None or rating == None:
+        return None
+
+    # add movie into dictionary, if all paramenter is valid value or truthy
+    elif title and genre and rating:
+        
+        movies["title"] = title
+        movies["genre"] = genre
+        movies["rating"] = rating
+        return movies
+    
+    # Otherwise, return None.
+    else:
+        return None

```

Comment:
It's best practice to use `is None` instead of `== None`. Also, we can make this slightly simpler by taking advantage of the fact that the function will exit once we hit a return statement:

```suggestion
    if title is None or genre is None or rating is None:
        return None

    # add movie into dictionary, if all paramenter is valid value or truthy
      movies["title"] = title
      movies["genre"] = genre
      movies["rating"] = rating
      return movies
```